### DISCUSSION TITLE
为什么我不能管理帐号啊

### DISCUSSION THREAD

**Discussion by cc1477** [qd-today/qd#544] [Q&A] [answered]
在 1panel 安装的，初次注册也登出后重新登录了

**Comment by a76yyyy** [comment, marked as answer]
> 在 1panel 安装的，初次注册也登出后重新登录了

可以进入容器bash后台执行命令设置账号管理权限

https://qd-today.github.io/qd/zh_CN/guide/deployment.html#%E8%AE%BE%E7%BD%AE%E7%AE%A1%E7%90%86%E5%91%98

**Reply by cc1477** [reply]
也试过了，还是不行

**Reply by a76yyyy** [reply]
> 也试过了，还是不行

清空浏览器缓存，或者换一个浏览器看看

**Reply by cc1477** [reply]
![IMG_20240825_144137.png](https://github.com/user-attachments/assets/18df2863-7104-453c-a25f-0ca2a7f359a6)

![IMG_20240825_144257.png](https://github.com/user-attachments/assets/7a73791f-111e-44e9-b25e-c47313eec95e)

![mmexport1724568195803.png](https://github.com/user-attachments/assets/b6a99d5c-5039-4409-8f73-f6991d973886)



还是不行诶，清了缓存，在手机上的浏览器也试过了，还是这样

**Reply by a76yyyy** [reply]
> ![IMG_20240825_144137.png](https://github.com/user-attachments/assets/18df2863-7104-453c-a25f-0ca2a7f359a6)
> 
> 
> 还是不行诶，清了缓存，在手机上的浏览器也试过了，还是这样

命令的最后一个参数是admin

**Reply by cc1477** [reply]
原来是这样我还以为是密码，感谢大佬的帮助
